cv19ResSupportV3.Tests/V3/E2ETests/UpdateHelpRequest.cs

```diff
@@ -77,6 +77,7 @@ public async Task UpdateHelpNeededFieldsUpdatesTheRecord()
                 HelpWithNoNeedsIdentified = null,
                 HelpWithAccessingSupermarketFood = true,
                 Id = requestObject.Id,
+                HelpNeeded = requestObject.HelpNeeded,
```

Comment:
It seems that all of the other changes were done against `CreateHelpRequest` endpoint _(based on the file names)_.

However, this change here is done on `UpdateHelpRequest.cs` file, which implies a change in `UpdateHelpRequest` endpoint. Does this change belong here?

Reply:
You're right, this is an unnecessary change made whilst fixing unit tests. I'll remove it.